tsconfig.json

```diff
@@ -23,7 +23,7 @@
     "src/**/*.tsx",
     "src/**/*.jsx",
     "src/**/*.js",
-    "src/custom.d.ts"
+    "@/types/*.d.ts"
```

Comment:
Could this change to just "src" and "@/types"

Reply:
Oh yeah, and the type-stuff should live in the @types-folder in projects root-level

Reply:
Done!